src/components/VmDetails/cards/UtilizationCard/DiskCharts.js

```diff
@@ -86,6 +90,7 @@ const DiskCharts = ({ vm, isRunning, id, ...props }) => {
           />
 
           {/* Disks don't have historic data */}
+          <EmptyBlock />
```

Comment:
Just use the style from `NoHistoricData` to avoid the extra component and css class and update the comment so it says why:

````jsx
  {/* Disks don't have historic data but stub the space so the card stretches like the others */}
  <div className={style['no-history-chart']} />
````